### PR TITLE
Add padding to the text below 767px on page : /docker-extension-meshery

### LINKED ISSUE #3324: No padding to the text on page: /docker-extension-meshery
#### Description
No padding to text on page: /docker-extension-meshery
Screenshot 2022-10-15 103903

#### Expected Behavior
Screenshot 2022-10-15 104221

#### Environment:
- Host OS: 
- Browser: 

---
<h3> Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by tinniaru3005**
**Description**
Fixed the issue by adding a padding of 15 px to left and right

This PR fixes #3324 
Closes #3324 
**Notes for Reviewers**
layer5

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Thanks, @tinniaru3005 🙂

File: src/sections/Docker-Meshery/dockerMeshery.style.js [hunk 1]
```diff
@@ -92,4 +92,8 @@ export const DockerMesheryWrapper = styled.div`
       padding: 5%;
       border-radius: 1rem;
     }
+    .jFTZLO {
```
Comment (Nikhil-Ladha) [inline src/sections/Docker-Meshery/dockerMeshery.style.js:95]: This won't work, as this is a temp/random className generated at the time of bundling.
Please use the actual className that the component has, or if needed add a new className to it.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 74c091e562db0be6bdd5767880001038ca6e6740 at: https://634cfaad47ed83287edf1214--layer5.netlify.app

**Comment by tinniaru3005** [comment]
Hey @Nikhil-Ladha, I made the changes. Can you please confirm if it looks fine now?

**Comment by tinniaru3005** [comment]
Hey @sahilsuman933, for the last commit I did it using VSCode. I hopw now it will work.
signoff

For the previous commits I was not aware of it. Do I need to close the PR, delete the branch, commit again and send the PR?

**Comment by leecalcote** [comment]
@tinniaru3005 you can follow these instructions to amend your sign-off to older commits - https://github.com/layer5io/layer5/pull/3331/checks?check_run_id=8943201532

**Comment by l5io** [comment]
🚀 Preview for commit 9dcbf730c4e3fda645cdcd14b63cb4a6b7810b2b at: https://634db04f6810a61683e51086--layer5.netlify.app

**Pull request closed without merging**